ampd/src/grpc/mod.rs

```diff
@@ -74,7 +77,13 @@ pub struct Server {
 impl Server {
     pub async fn run(self, token: CancellationToken) -> Result<(), Error> {
         let addr = SocketAddr::new(self.config.ip_addr, self.config.port);
+        let trace_layer = trace::TraceLayer::new_for_grpc()
+            .make_span_with(trace::DefaultMakeSpan::new().level(tracing::Level::INFO))
```

Comment:
the documentation on this is pretty lacking unfortunately. I had to dig into the source code to understand what it's doing. Could you add a comment about what this span adds to the log, so nobody will think this is superfluous and remove it in the future?